agent/utils/utils.go

```diff
@@ -199,3 +200,11 @@ func ParseBool(str string, default_ bool) bool {
 	}
 	return res
 }
+
+// IsErrorCodeEqual returns true if the err implements Error
+// interface of awserr and it has the same error code as
+// the passed in error code.
+func IsErrorCodeEqual(err error, code string) bool {
```

Comment:
Since this method also checks if this is an `awserr`, can you name this `IsAWSErrorCodeEqual`?

Reply:
ok